mne/io/tests/test_raw.py

```diff
@@ -153,3 +161,77 @@ def test_annotation_property_deprecation_warning():
     assert len(w) is 0
     with pytest.warns(DeprecationWarning, match='by assignment is deprecated'):
         raw.annotations = None
+
+
+@pytest.fixture
+def _dummy_raw():
+    info = create_info(ch_names=10, sfreq=10., ch_types=None, montage=None,
+                       verbose=None)
```

Comment:
`info = create_info(10, 10.)` is good enough here, `ch_names=10, sfreq=10.` if you prefer `kwargs`. The other stuff adds clutter more than it aids understanding. It's generally a good idea to use a DATA ch_type such as `eeg`, too, so this usually shows up in tests simply as:
```
info = create_info(10, 10., 'eeg')
```